lib/parse-args.js

```diff
@@ -154,12 +158,6 @@ function buildYargs (withCommands = false) {
     })
     .pkgConf('c8')
     .demandCommand(1)
-    .check((argv) => {
```

Comment:
From my understanding, this function should not actually change arguments but only checks to see if they are valid.